### DISCUSSION TITLE
Why there are automatically generated "IRU64HHN" note

### DISCUSSION THREAD

**Discussion by R3c0ger** [windingwind/zotero-better-notes#920] [Q&A] [answered]
![图片](https://github.com/windingwind/zotero-better-notes/assets/116271028/c115c4e2-d75e-4a98-9637-c0247a31e3df)
I can't even figure out the regularity of how were these notes generated.
平白无故时不时生成一些IRU64HHN笔记，我甚至找不出自动生成这些笔记的时间规律……望各位大佬指教，感激不尽！

**Comment by windingwind** [comment, marked as answer]
与本插件无关。禁用类似chartero或style插件

**Reply by R3c0ger** [reply]
好的，感谢感谢，效果拔群
